content/root/quick-start-guide.textile

```diff
@@ -489,6 +489,7 @@ This guide introduced the basic concepts of Ably and illustrated how easy it is
 * Find out more about the "Realtime Client Library SDK":/realtime.
 * Find out more about the "REST Client Library SDK":/rest.
 * Try out some "tutorials":https://ably.com/tutorials.
+* Provision Ably applications using the "Control API":/control-api.
```

Comment:
Would it be worth changing the absolute ref to tutorials here, too?

Reply:
> Provision Ably applications

Minor, but can we change to "Ably apps" for consistency in Control API docs. Thanks.

Reply:
Changed both, thanks!